coremltools/converters/mil/frontend/torch/test/testing_utils.py

```diff
@@ -123,28 +125,32 @@ def _convert_to_inputtype(inputs):
     )
 
 
-def generate_input_data(input_size, rand_range=(0, 1), torch_device=torch.device("cpu")):
+def generate_input_data(
+    input_size, rand_range=(0, 1), dtype=np.float32, torch_device=torch.device("cpu")
```

Comment:
when adding a new argument to an existing function,
it is always better to "append it" in the end,
for the backward compatibility.

Reply:
Good point! Luckily this time everything is good I think
1. CI is green, so nothing is broken inside our test
2. This is not a public API, so nothing outside should get affected

Reply:
For future reference still I think it is important to follow the correct coding :)

Reply:
I am good for this time we don't address it.